### ISSUE TITLE
is there any plan for quilljs2.0

### ISSUE THREAD

**Issue by fred8617** [zenoamaro/react-quill#601]
Thanks for filing an issue! 

If at all possible, please provide a Codepen to demonstrate the problem you're having with React Quill. Here's a [template] to get you started.

[template]: 
https://codepen.io/pen?template=xgyOXQ

### FAQ

some module like better-table need at least quilljs 2.0

**How do I access the wrapped Quill instance?**
See the [instance methods] and [API] documentation.

[instance methods]:
https://github.com/zenoamaro/react-quill#methods

[API]:
https://github.com/zenoamaro/react-quill#api-reference

**Is there a bug in Quill?**
Please refer to the [Quill issues page].

[Quill issues page]: 
https://github.com/quilljs/quill/issues

#### React-Quill version 
- [ ] master
- [ ] 0.4.1
- [ ] 1.0.0-beta-1
- [ ] 1.0.0-beta-2
- [ ] 1.0.0-beta-3
- [ ] 1.0.0-beta-4
- [ ] 1.0.0-beta-5
- [ ] 1.0.0-rc-1
- [ ] 1.0.0-rc-2
- [ ] 1.0.0-rc-3
- [ ] 1.0.0
- [ ] 1.1.0
- [ ] Other (fork)

**Comment by robphoenix** [comment]
To clarify, will `react-quill` support quilljs v2? AFAICT it's needed to be able to use https://github.com/soccerloway/quill-better-table

**Comment by xeger** [comment]
I am in the exact same position as @robphoenix: I'd like to make use of tables (either those built in to Quill 2, or the "better" module) but I'm also very much a fan of ReactQuill.

@zenoamaro is a Quill 2 upgrade something you've investigated at all? I understand that Quill 2 is radically different from 1.x, so if your answer is "haven't even looked at it," I understand!

**Comment by alexkrolick** [comment]
We'll support it when it's stable; right now it is in beta

**Comment by robphoenix** [comment]
grand, thanks @alexkrolick

**Comment by christopherhuii** [comment]
Is there a targeted release date for v2? I'm currently doing some research around rich text editors and I'm uncertain whether I should dive into beta or stick with the latest stable release. A rough estimate would help me decide which version to adopt.

**Comment by lukevp** [comment]
It's been 2 years since quill 2.0 started development, it would be nice if react-quill could support it at least as a beta version of react-quill.  I'm guessing a lot of people are using the quill 2.0 beta already in production since the 1.0 version doesn't support tables.

**Comment by JoeCMorgan** [comment]
Has anyone found _any_ suitable work around for using react-quill with tables?

**Comment by steodor** [comment]
Any updates?

**Comment by prasanthLalapeta** [comment]
Any updates ?

**Comment by sachirjn** [comment]
Any updates?

**Comment by andrew2558** [comment]
Any updates?

**Comment by xeger** [comment]
As a reminder, QuillJS is a separate product that @zenoamaro doesn't maintain; for updates on Quill 2,0, you can refer to [this tracking issue](https://github.com/quilljs/quill/issues/3251) created by users.

Long story short, development is stalled, seemingly due to lack of resources and/or business conflicts of interest. There are some discussions of forking the project but no earnest efforts.

At this point, it's best to plan on Quill 2.0 never being released, and to plan your own software dev accordingly: use Quill 1, but build façades that will make it easier to replace if needed.

**Comment by juanlanus** [comment]
ReactQuill 2 is here, baby!

I'm about to switch to react-quill and I wonder if I should go directly for the beta, should I?  
I don't mind if there are few bugs as long as it's fairly stable.  

What do you recommend, what's the status of the beta?

**Comment by growthwp** [comment]
@juanlanus Sorry, how do you mean ReactQuill 2 is here? Can you link to a release link or something?

**Comment by juanlanus** [comment]
@growthwp
Open https://github.com/zenoamaro/react-quill#quick-start and scroll back one page

**Comment by tmilewski** [comment]
@juanlanus That hasn't been updated for 14 months.

**Comment by minspy** [comment]
> @juanlanus That hasn't been updated for 14 months.

More than, for now is been 19 months.

**Comment by casamia918** [comment]
I'm dispaired. 

At first, I installed quilljs v1.3.7. 

An then, we should use [quill-better-table](https://github.com/soccerloway/quill-better-table/) library to support table feature. But this library only support on quilljs 2.0.0-dev. So I update quill version. 
And then, I encountered criticall error of quill-better-table. This error can be solved by [this folked library](https://github.com/LosEcher/quill-better-table)
And then, still the copy and paste from my excel into quill is not working. Then I found that react-quill has quilljs v.1.3.7 dependencies. 
And then, I found this issue and surprised by the date of this issue date...

**Comment by JoeCMorgan** [comment]
Honestly I’d forget about Quill and use TinyMCE - never had any problems with it since moving over

**Comment by casamia918** [comment]
@JoeCMorgan 
After seeing your comment, I also changed to TinyMCE. Then, all of my problems are gone.

**Comment by herzuull** [comment]
Yes, but TinyMCE is freemium, and can be complicated if you want to use image management, I read that you need an instance of tinyMCE to handle image upload, am I wrong ?

**Comment by JoKenPo** [comment]
Will this be released? :(

**Comment by Prabhakar-Poudel** [comment]
could be closed now that we have v2?
https://www.npmjs.com/package/react-quill/v/2.0.0

**Comment by gbenavid** [comment]
@Prabhakar-Poudel it seems this is still an issue since react-quill V2 is still dependent on quill ^1.3.7
https://github.com/zenoamaro/react-quill/blob/master/package.json#L48

**Comment by Ali-Chimim** [comment]
Is this still maintained ?

**Comment by KalanaPerera** [comment]
any suggestions? I was checking Lexical, single editor works fine.

having few problems when using multiple editors on the same page.

**Comment by hamidouikene** [comment]
Hi all,
We ar eusing react quill in one of our projects and it looks like Chrome is going to stop supporting mutation events in few months and we will probably endup with the editor not working. 
Could you please tell us if you are planning to migrate to quillJS v2?

Thank you

**Comment by Sashkko** [comment]
Up! This is also a relevant question for me!

**Comment by eivindszalay** [comment]
> Hi all, We ar eusing react quill in one of our projects and it looks like Chrome is going to stop supporting mutation events in few months and we will probably endup with the editor not working. Could you please tell us if you are planning to migrate to quillJS v2?
> 
> Thank you

Relevant for our team as well.

**Comment by t-ashraf** [comment]
See this PR for upgrading to Quill 2.0: https://github.com/zenoamaro/react-quill/pull/967

**Comment by GAumala** [comment]
There is no need to update this library for Quill 2.0.

You can add Quill 2.0 directly to your React project without much hassle. The Quill webiste has a neat playground for [Using Quill with React.](https://quilljs.com/playground/react)

I switched today and it took me less than 2 hours to get it working just like before.

**Comment by niraj2477** [comment]
>There is no need to update this library for Quill 2.0.

Why not? If it's updated in package , no one has to manually do it right?
